pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -17,8 +17,7 @@ namespace {
 // we're going to be processing with std::next_permutation() --
 // e.g. if we used pointers as comparison values, our order of
 // iteration could vary run-to-run, creating non-determinism.
-template <typename V>
-class Orderer {
+template <typename V> class Orderer {
```

Comment:
What version of clang-format are you using? We need to use version 9.

Reply:
I will get version 9, thanks.

Reply:
I couldn't figure out how to run clang-format on *just* my changes so I did a `git reset` and recommitted all changes (with clang-format-9 formatting) as a single commit, force pushed.